jumpscale/packages/vdc/chats/new_vdc.py

```diff
@@ -15,6 +15,8 @@
 class VDCDeploy(GedisChatBot):
     title = "VDC"
     steps = ["vdc_info", "storage_farms_selection", "deploy", "initializing", "success"]
+    VDC_INIT_WALLET_NAME = "vdc_init"
+    GRACE_PERIOD_WALLET_NAME = "grace_period"
```

Comment:
```suggestion
    VDC_INIT_WALLET_NAME = j.config.get("VDC_INITIALIZATION_WALLET", "vdc_init")
    GRACE_PERIOD_WALLET_NAME = j.config.get("GRACE_PERIOD_WALLET", "grace_period")
```

Maybe we can try to get from config manager first, as we allow that option, if user wants to change